scenarios/kubernetes_build.py

```diff
@@ -71,6 +78,21 @@ def main(args):
     else:
         check('make', 'release')
     check('../release/push-build.sh', *push_build_args)
+    pull_refs = os.getenv('PULL_REFS', '')
+    if pull_refs:
+        # log push-build location to path child jobs can find
+        # (gs://<shared-bucket>/$PULL_REFS/bazel-build-location.txt)
+        # first get build location the same way push-build.sh does
+        bucket = args.release or 'kubernetes-release-dev'
+        dest = 'ci'
+        if args.suffix:
+            dest += args.suffix
+        build_tar = tarfile.open("_output/release-tars/kubernetes.tar.gz")
+        latest = build_tar.getmember("kubernetes/version").read().strip()
+        gcs_build = os.path.join('gs://', bucket, dest, latest)
```

Comment:
duplicating the logic of push-build.sh seems pretty fragile... I'd rather either we remove that logic from push-build.sh or add an option to push-build.sh to create this pointer file. @david-mcmahon